devices/GnssDevice/Tests/GnssDeviceTests/NMEA0183ParserTests.cs

```diff
@@ -26,7 +26,7 @@ public void ParseGngsa(string command, byte expectedMode, byte expectedFix)
         }
 
         [TestMethod]
-        [DataRow("$GPGLL,5109.0262317,N,11401.8407304,W,202725.00,A,D*79", 51.1504372f, -114.03067884f)]
```

Comment:
Maybe add new DataRow instead of editing existing?

Reply:
if you have real ones, give them to me :-) Happy to add more! . Same for next one.
I just bought a module to run more tests as well.

Reply:
Hmm, I thought It was real from my tests... Nevermind then, If anythink breaks durring my tests of real GPS device, I'll add more tests.